### ISSUE TITLE
HEMCO.log file gets closed, reopened as fort.11

### ISSUE THREAD

**Issue by TerribleNews** [geoschem/GCHP#23]
I am working off of version 12.5 in the old repo, but I found the source of the problem and that code is still in the gchp_ctm repo so I thought I'd drop this bug report here. The symptoms may be different in gchp_ctm, but I'll post what happens to my runs here. 

Since the HEMCO.log file is the first persistent output filed opened, it gets a unit number of 11 from findFreeLUN. Then, at some point after EMISSIONS_INIT is finished, MAPL_cfio/ESMF_CFIOMod.F90::ESMF_CFIOFileOpen is called and these lines are executed:
```
    858       open(11, file=fileName)
    859       read(11, '(a)') dset
    860       close(11)
```
This closes my HEMCO.log file handle. Next time HCO_MSG gets called, it opens a new default file name, which is fort.11, and the HEMCO log output continues in that file. It's obviously not critical, but it is annoying and should be easy to fix. For my code, because I couldn't access inquireMod from the MAPL_io folder, I've just added a for loop searching for a free LUN before that and replaced the 11 with that LUN variable. I don't know what the most copacetic fix is for the new repo.

**Comment by lizziel** [comment]
I believe this issue was fixed in 12.7 by @msulprizio. The problematic code that was driving the issue was in hco_readlist_mod.F90 (and likely other places). HCO_MSG was called with:
`CALL HCO_MSG( MSG )`
but should be called with
`CALL HCO_MSG(HcoState%Config%Err,MSG)`
to write the HEMCO message to the correct location (HEMCO.log).

I have not seen the fort.11 pop up since so I believe this solution is robust.

**Comment by lizziel** [comment]
I should add that this is for the standard model. If you have custom code that taps into MAPL and/or alters HEMCO it's entirely possible it popped up again. But I'd suggest updating to a newer version before diving into figuring out if this is the case.
